docs/reference/infrastructure/feedstocks.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Feedstocks
+---
+
+# Feedstocks and other package-building repositories
+
+Most packages in conda-forge come from a repository named `<package_name>-feedstock`, but a very small subset comes from specific, non-feedstock, repositories.
+
+## Feedstocks
+
+- ⚙️ Deployed in Github repositories
+- 🔒 Has access to Azure Pipelines, Github Actions, Anaconda.org (cf-staging)
+- 🔐 Might have access to Travis CI, Cirun via `admin-requests` (WIP)
+- 🤖 Integrated with [`admin-migrations`](./automated-maintenance.md#admin-migrations), [`admin-requests`](./automated-maintenance.md#admin-requests), [`autotick-bot`](./automated-maintenance.md#autotick-bot), [`webservices`](./automated-maintenance.md#webservices).
+
+conda-forge has thousands of feedstocks.
+Each feedstock hosts a recipe plus the required pipelines, supporting scripts and configuration metadata.
+
+The contents of a feedstock are well specified. Only two locations are user-managed:
+
```

Comment:
maybe adding a general tree of contents for a feedstock here?
```
name
   \a
   \b...
```

Reply:
or at the end, to give a view of the organization of the repo?

Reply:
I feel adding more details belongs in a (currently inexistent) conda-smithy documentation maybe?